From b7aed472c6639e2a523d0ef90dba61246d7bd9b4 Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Mon, 25 Jul 2016 12:32:02 -0400
Subject: [PATCH] rtl8xxxu: Enable 8188eu driver

This enables the 8188eu driver - this should work by now.

Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---
 drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
+++ b/drivers/net/wireless/realtek/rtl8xxxu/rtl8xxxu_8188e.c
@@ -539,7 +539,6 @@ static int rtl8188eu_load_firmware(struc
 
 	ret = rtl8xxxu_load_firmware(priv, fw_name);
 
-	return -EINVAL;
 	return ret;
 }
 
